.github/workflows/constraints.txt

```diff
@@ -1,5 +1,8 @@
-pip==21.1.3
+pip<=20.3
```

Comment:
Is this <=20.3 coming from home assistant?

Reply:
Yes it was